### PR TITLE
Feat/container layout recipe details

### PR DESCRIPTION
**Pull request by Arjay84**

### CODE REVIEW

File: components/recipeDetails/recipeDetails.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ export interface RecipesDetailsProps {
 const RecipesDetails: React.FC<RecipesDetailsProps> = ({ recipe }) => {
   return (
     <div>
-      <h1 className="text-2xl font-bold mb-4">{recipe.name}</h1>
+      <h1 className="text-5xl font-bold mb-4">{recipe.name}</h1>
```
Comment (moncapitaine) [inline components/recipeDetails/recipeDetails.tsx:13]: max h2, h1 is reserved for overall headlines.

File: components/recipeDetails/recipeDetails.tsx [hunk 2]
```diff
@@ -24,17 +24,17 @@ const RecipesDetails: React.FC<RecipesDetailsProps> = ({ recipe }) => {
         />
       </div>
 
-      <div id="recipe-author" className="mb-4">
+      <div id="recipe-author" className="font-bold text-sm mb-4">
```
Comment (moncapitaine) [inline components/recipeDetails/recipeDetails.tsx:27]: remove all id attributes. They are not used anywhere...

File: components/recipeDetails/recipeDetails.tsx [hunk 3]
```diff
@@ -24,17 +24,17 @@ const RecipesDetails: React.FC<RecipesDetailsProps> = ({ recipe }) => {
...
         Author, Release Date
       </div>
 
       <div id="recipe-description" className="mb-4">
-        <h2 className="text-xl font-bold mb-4">Description</h2>
+        <h2 className="text-3xl font-bold mb-4">Description</h2>
```
Comment (moncapitaine) [inline components/recipeDetails/recipeDetails.tsx:32]: h3

File: components/recipeDetails/recipeDetails.tsx [hunk 4]
```diff
@@ -24,17 +24,17 @@ const RecipesDetails: React.FC<RecipesDetailsProps> = ({ recipe }) => {
...
         <p className="text-small mt-2">{recipe.description}</p>
       </div>
 
       <div id="ingredients-container" className="mb-4">
-        <h2 className="text-xl font-bold mb-4">Ingredients:</h2>
+        <h2 className="text-3xl font-bold mb-4">Ingredients:</h2>
```
Comment (moncapitaine) [inline components/recipeDetails/recipeDetails.tsx:37]: h3

File: components/recipeDetails/recipeDetails.tsx [hunk 5]
```diff
@@ -24,17 +24,17 @@ const RecipesDetails: React.FC<RecipesDetailsProps> = ({ recipe }) => {
...
         {recipe.ingredients?.map((ingredient, index) => (
```
Comment (moncapitaine) [inline components/recipeDetails/recipeDetails.tsx:38]: flex layout.

**Review by DexHarris — APPROVED** [review]

**Review by pSchwietzer — APPROVED** [review]

**Pull request merged**